cranelift/codegen/src/isa/x64/abi.rs

```diff
@@ -514,7 +543,19 @@ impl ABIMachineSpec for X64ABIMachineSpec {
                     ));
                     cur_offset += 8;
                 }
-                _ => unimplemented!(),
+                RegClass::V128 => {
+                    if cur_offset % 16 == 8 {
+                        cur_offset += 8;
+                    }
```

Comment:
Perhaps we should `debug_assert` or something here that we are still aligned?

Reply:
Did one better; just used alignment helper.